frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/components/assisted-installer/utils.ts

```diff
@@ -322,7 +322,10 @@ export const useClusterDeployment = ({
 }): CIM.ClusterDeploymentK8sResource | undefined => {
     const [clusterDeployments] = useRecoilValue(waitForAll([clusterDeploymentsState]))
     return useMemo(
-        () => clusterDeployments.find((cd) => cd.metadata.name === name && cd.metadata.namespace === namespace),
+        () =>
```

Comment:
Is this needed? ``Array.find()` already returns `undefined` if no match is found.

Reply:
Right. That was meant as an optimization, since the `name === undefined` is a frequent case, the array does not need to be searched.